src/components/EditFeatureDrawer/EditFeatureFullForm/index.tsx

```diff
@@ -93,7 +93,7 @@ export const EditFeatureFullForm: React.FC<EditFeatureFullFormProps> = ({
 
     const updatedFeatures = await executeGetFeature({
       layer: layer,
-      filter: equalTo('id', id)
+      filter: equalTo('ogc_fid', id)
```

Comment:
Actually both values aren't quite sophisticated since the id column can be any numeric field of the feature type. We should think about making it configurable or getting it dynamically from the `DescribeFeatureType`.